modules/ROOT/pages/amazon/amazon-dynamodb-connector-studio.adoc

```diff
@@ -0,0 +1,153 @@
+=== Basic Authentication
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
+
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Name |Enter the configuration name.
+|Connection | Select `Basic`.
+|Session Token | Optionally enter the session token provided by Amazon Security Token Service (STS).
+|Access Key | Enter the access key provided by Amazon.
+|Secret Key | Enter the secret key provided by Amazon.
+|Try Default AWS Credentials Provider Chain | Set to `true` to obtain credentials from the AWS environment.
+|Region Endpoint | Select the region endpoint for the service.
+|===
+
+The following screenshot shows an example of configuring Basic authentication:
+
+.Basic authentication fields
+image::amazon/amazon-dynamodb-basic-authentication.png[To configure authentication, select `Basic` in the *Connection* field and then complete the fields on the *General* tab.]
+
+The first item shows where to specify that the connector will use Basic authentication. The second item shows the *General tab*, which contains fields related to Basic authentication.
+
+[[role-authentication]]
+=== Role Authentication
+
+Enter the following information on the *General* tab of the global element configuration screen to configure Role authentication:
+
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Name |Enter the configuration name.
+|Connection | Select `Role`.
+|Role ARN | Enter the role to assume to gain cross-account access.
+|Access Key | Enter the access key provided by Amazon.
+|Secret Key | Enter the secret key provided by Amazon.
+|Try Default AWS Credentials Provider Chain | Set to `true` to obtain credentials from the AWS environment.
+|Region Endpoint | Select the region endpoint for the service.
+|===
+
+The following screenshot shows an example of configuring Role authentication:
+
+.Role authentication fields
+image::amazon/amazon-dynamodb-role-authentication.png[To configure Role authentication, select `Role` in the *Connection* field and then complete the fields on the *General* tab.]
+
+The first item shows where to specify that the connector will use Role authentication. The second item shows the *General tab*, which contains fields related to Role authentication.
+
+[[configure-other-fields]]
+== Configure the Other Connector Fields
+
+After you configure a global element for Amazon DynamoDB connector, configure the other required fields for the connector:
+
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+|Table name | Name of the table to create
+|Attribute definitions | Attributes that describe the key schema for the table and its indexes
+|Key schemas | Attributes that compose the primary key for a table or index
+|===
+
+== See Also
+
+* xref:introduction/introduction-to-anypoint-connectors.adoc[Introduction to Anypoint Connectors]
+* xref:introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]
```

Comment:
Do we really need to reference the generic version of the same topic as this one?